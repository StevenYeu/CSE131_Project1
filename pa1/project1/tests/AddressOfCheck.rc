function : void main(){
    int x;
    int y;
    int *z;
    const int w = 77;
    z = &x;
    &x = nullptr;
    y = *&x;
    *&x = y;
    *&w = y;

    &*z = z;
    int u = &1;

}
