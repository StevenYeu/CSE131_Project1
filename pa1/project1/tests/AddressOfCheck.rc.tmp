Error, "project1/tests/AddressOfCheck.rc": 
  Left-hand operand is not assignable (not a modifiable L-value).
Error, "project1/tests/AddressOfCheck.rc": 
  Left-hand operand is not assignable (not a modifiable L-value).
Error, "project1/tests/AddressOfCheck.rc": 
  Non-addressable argument of type int to address-of operator.
Compile: failure.
