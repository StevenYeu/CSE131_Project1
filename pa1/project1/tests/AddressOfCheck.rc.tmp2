  Left-hand operand is not assignable (not a modifiable L-value).
  Left-hand operand is not assignable (not a modifiable L-value).
  Non-addressable argument of type int to address-of operator.
Compile: failure.
