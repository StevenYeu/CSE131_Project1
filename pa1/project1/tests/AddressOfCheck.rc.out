Error, "/tmp/cs131_testrunner_KI0qMF/file.rc": 
  Left-hand operand is not assignable (not a modifiable L-value).
Error, "/tmp/cs131_testrunner_KI0qMF/file.rc": 
  Left-hand operand is not assignable (not a modifiable L-value).
Error, "/tmp/cs131_testrunner_KI0qMF/file.rc": 
  Non-addressable argument of type int to address-of operator.
Compile: failure.
